'

```diff
@@ -0,0 +1,320 @@
+var winston = require('winston');
+var routeUtils = require('../api/routes/utils.js');
+var utils = require('../bin/utils.js');
+
+var Category = require('../api/models/Category.js');
+var Transaction = require('../api/models/Transaction.js');
+var User = require('../api/models/User.js');
+
+describe('Job utils: ', function() {
+  var cb;
+  beforeEach(function() {
+    cb = jasmine.createSpy().andReturn();
+    spyOn(winston, 'log').andCallFake(function(arg1, arg2, arg3, arg4) {
+      return;
+    });
+  });
+
+  afterEach(function() {
+    expect(cb.callCount).toEqual(1);
+  });
+
+  describe('migratePercentagesAndDividends: ', function() {
+    var expert, investor1, investor2;
+    beforeEach(function() {
+      expert = {
+        _id: '123',
+        portfolio: [],
+        categories: [
+          {
+            name: 'Coding',
+            reps: 20,
+            investors: [
+              { id: '456' },
+              { id: '789' },
+            ]
+          }
+        ],
+        save: jasmine.createSpy().andReturn()
+      };
+
+      investor1 = {
+        _id: '456',
+        reps: 1,
+        portfolio: [
+          {
+            category: 'Coding',
+            investments: [ { userId: '123', percentage: 100, amount: 5 } ]
+          }
+        ],
+        save: jasmine.createSpy().andReturn()
+      };
+
+      investor2 = {
+        _id: '789',
+        reps: 3,
+        portfolio: [
+          {
+            category: 'Coding',
+            investments: [ { userId: '123', percentage: 75, amount: 15 } ]
+          }
+        ],
+        save: jasmine.createSpy().andReturn()
+      };
+
+      users = [expert, investor1, investor2];
+    });
+
+    it('handles error finding users', function() {
+      spyOn(User, 'find').andCallFake(function(callback) {
+        return callback('Error', null);
+      });
+      utils.migratePercentagesAndDividends(cb);
+      expect(winston.log).toHaveBeenCalledWith('error',
+        'Error migrating percentages and dividends: %s', 'Error');
+      expect(users[1].reps).toEqual(1);
+      expect(users[2].reps).toEqual(3);
+    });
+
+    it('divides percentages by 100 and adds accurate dividends', function() {
```

Comment:
I added tests in here for the new migrations. I think git gets confused because all the unit tests look the same